### DISCUSSION TITLE
Firebase auth

### DISCUSSION THREAD

**Discussion by jeffersonsolorzano** [Zerosubnet/directus-extension-external-jwt#10] [Q&A] [answered]
Hi, i was wondering if a firebase JWT might work?, is there any way to configure this extension to make a JWT generated by firebase work as a valid token for directs

**Comment by jmdodge** [comment]
I was hoping to learn this as well.

**Reply by jeffersonsolorzano** [reply, marked as answer]
https://github.com/workatease/directus-extension-firebase-user-auth/blob/master/src/index.ts 

Check this extension:

**Reply by jmdodge** [reply]
This looks perfect, thank you @jeffersonsolorzano !

May I ask, as I'm new to the Directus ecosystem - how does one normally find these extensions? Is there any registry or central organization that lists them? I've seen some community spaces but nothing that looks particularly organized in this regard.

**Reply by jeffersonsolorzano** [reply]
No, you have to find it by yourself.

**Reply by jmdodge** [reply]
Gotcha. Thank you for taking the time to respond!
